llvm/lib/Target/EraVM/MCTargetDesc/EraVMMCCodeEmitter.cpp

```diff
@@ -60,15 +66,66 @@ class EraVMMCCodeEmitter : public MCCodeEmitter {
                         SmallVectorImpl<MCFixup> &Fixups,
                         const MCSubtargetInfo &STI) const;
 
+  uint64_t fixOpcode(const MCInst &MI, uint64_t EncodedInstr,
+                     const MCSubtargetInfo &STI) const;
+
   void encodeInstruction(const MCInst &MI, raw_ostream &OS,
                          SmallVectorImpl<MCFixup> &Fixups,
                          const MCSubtargetInfo &STI) const override;
+
+private:
+  uint64_t getRegWithAddend(const MCInst &MI, unsigned BaseReg,
+                            int Addend) const;
 };
 
+int modeEncodingByMarker(const MCOperand &Op) {
```

Comment:
Can it be `static`?

Reply:
Sure. I thought it is inside an anonymous namespace, but it is not. Fixed in 2d74a1a6e5e84a592b0bcd1ff8480abd17f4e391.